CODEOWNERS

```diff
@@ -40,10 +40,4 @@
 /test/integration/content-picker/           @box/ui-elements
 /test/integration/content-preview/          @box/preview
 /test/integration/content-sidebar/          @box/preview
-/test/integration/content-uploader/         @box/create-communicate
-
-# Features
-/src/elements/content-sidebar/activity-feed/comment/    @box/collab @box/preview
```

Comment:
removed this section because collab is no longer a team and the preview team already is the codeowner of the `/src/elements/content-sidebar`